applications/StructuralMechanicsApplication/custom_elements/base_solid_element.cpp

```diff
@@ -1696,8 +1696,13 @@ void BaseSolidElement::CalculateLumpedMassVector(
     // LUMPED MASS MATRIX
     const double total_mass = GetGeometry().DomainSize() * density * thickness;
 
-    Vector lumping_factors;
-    lumping_factors = GetGeometry().LumpingFactors( lumping_factors );
+    Vector lumping_factors(number_of_nodes);
+    if (this->Has(LUMPING_FACTORS)) {
+        lumping_factors = this->GetValue(LUMPING_FACTORS);
+    } else {
+        lumping_factors = GetGeometry().LumpingFactors( lumping_factors );
+        this->SetValue(LUMPING_FACTORS, lumping_factors);
```

Comment:
See @philbucher, a const_cast is needed and I don't like that

Reply:
well the element shouldnt set it
if it would set it here the first time it is called it wouldn't work in restarts

Reply:
this is an enrmous amount of memory!